src/core/src/bootstrap/Bootstrapper.py

```diff
@@ -174,4 +174,3 @@ def check_sudo_status(self, raise_if_not_sudo=True):
                                             "Exception details: " + str(exception))
             if raise_if_not_sudo:
                 raise
-
```

Comment:
Revert -- please catch things like this with a review in GitHub after pushing iterations.